integration/pkg/api/osrm/table/mock_response.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/Telenav/osrm-backend/integration/pkg/api/osrm/osrmtype"
 )
 
+// 1 * 4
```

Comment:
Does the file for test only? use the file name `mock_response_test.go` instead, so it will not be compiled into production code.